core/src/main/java/com/backblaze/b2/json/B2JsonOptions.java

```diff
@@ -47,18 +47,36 @@
      *
      * When set, fields marked as @B2Json.sensitive will be redacted from the serialized
      * JSON and replaced with the string value "***REDACTED***"
-     * The output will be valid Json but the structure/types will not conform to the expected
+     * The output will be valid Json but the structure/ty
+     * pes will not conform to the expected
      * output. Use for logging situations where round-tripping the JSON is not required
      */
     private final boolean redactSensitive;
 
+    /**
+     * How to format the serialized string
+     *
+     * PRETTY is the default and produces indented, multi-line JSON
+     * COMPACT produces a single line of text with no optional whitespace
+     */
+    public enum SerializationOption {
```

Comment:
I keep thinking this might "JsonFormatOption" or "OutputFormatOption", "FormatOption" or...I don't know.  :)
since i don't have a definitive suggestion of what to make better, i'm fine leaving it as is.  :)